### ISSUE TITLE
CategoricalAxis() not exported

### ISSUE THREAD

**Issue by dpabon** [JuliaDataCubes/YAXArrays.jl#288]
In v0.5

```CategoricalAxis()``` is not defined (maybe not exported?)

**Comment by felixcremer** [comment]
This is part of the change to DimensionalData.jl. We don't have our own Axis types like RangeAxis and CategoricalAxis anymore. Everything is now a DimensionalData.Dim and you should replace your previous `CategoricalAxis("AxisName", ["Val1", "Val2"])` by the Dimension `Dim{:AxisName}(["Val1", "Val2"])`.

**Comment by dpabon** [comment]
Hi Felix,

Thanks for the clarification. As this version YAXArrays break previous code. I will suggest mentioning it in the release notes. Maybe also to include which functions are not exported anymore and the equivalent.

**Comment by felixcremer** [comment]
Yes, that is a good idea. Would you like to start a PR for that?

**Comment by dpabon** [comment]
I don't have time this week, but I can do it after my holidays in mid-August.

**Comment by dpabon** [comment]
I saw this is now included in the documentation, then I will close this issue.
